content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -93,6 +93,44 @@ EndpointSlices support three address types:
 * IPv6
 * FQDN (Fully Qualified Domain Name)
 
+### Conditions
+
+The EndpointSlice API stores conditions about endpoints that may be useful for consumers.
+The three conditions are `ready`, `serving`, and `terminating`.
+
+#### Ready
+
+`Ready` is a condition that maps to a pod's `Ready` condition. A running pod with the `Ready` condition set to `True`
```

Comment:
Nit: Elsewhere in this doc, Kubernetes resource names like "pod" are capitalized as "Pod". I'm not actually sure if that's the standard practice though, so if not, feel free to disregard. Similarly, `True` is capitalized here but is not elsewhere. Not sure what the correct value is, but should be consistent.

Reply:
One more tiny nit - the rest of this file is being wrapped to 80 char line lengths, definitely not required but would be cool if we could keep that pattern up.

Reply:
After checking the source, I don't think we have condition named `Ready`. The condition name is `ready`, `serving` and `terminating`.

We use `true` for boolean values rather than `True`.

Reply:
good points, I keep forgetting to use serialized json fields in docs :facepalm: